ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -565,3 +512,35 @@ def check_capacity_breakdown(self, project_name, pod_name):
             logger.error(f"The pod {pod_name} not found on capacity_breakdown")
             res = False
         return res
+
+    def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
```

Comment:
Why did you add `-> tuple`?

Reply:
it returns tuple compression_status_blockpools_tab, compression_status_blockpools_details